pkg/container/container.go

```diff
@@ -121,6 +121,18 @@ func (c *containerModule) ensureNamespace(ctx context.Context, client *container
 	return service.Create(ctx, namespace, nil)
 }
 
+func removeRunMount() oci.SpecOpts {
```

Comment:
Can we change the "seed" spec we use instead of removing the run mount ? just wondering. 
Otherwise, i think this is a good solution.

Reply:
I would love to and there is an open issue about that for quite some time now: https://github.com/threefoldtech/zos/issues/19
But for some reason no progress has ever been made on that topic